torch/csrc/api/src/nn/modules/activation.cpp

```diff
@@ -146,6 +146,21 @@ Tensor LogSoftmaxImpl::forward(const Tensor& input) {
 
 // ============================================================================
 
+Softmax2dImpl::Softmax2dImpl() {}
```

Comment:
We can remove the default constructor here :D

Reply:
OK, I fixed it.